src/codegen/parser.cpp

```diff
@@ -1150,50 +1084,25 @@ static std::vector<char> _reparse(const char* fn, const std::string& cache_fn, A
     file_data.insert(file_data.end(), (char*)&checksum, (char*)&checksum + CHECKSUM_LENGTH);
     checksum = 0;
 
-    if (ENABLE_CPYTHON_PARSER || ENABLE_PYPA_PARSER || inherited_flags) {
-        if (ENABLE_CPYTHON_PARSER) {
-            FileHandle fp(fn, "r");
-            PyCompilerFlags cf;
-            cf.cf_flags = inherited_flags;
-            ArenaWrapper arena;
-            assert(arena);
-            mod_ty mod = PyParser_ASTFromFile(fp, fn, Py_file_input, 0, 0, &cf, NULL, arena);
-            if (!mod)
-                throwCAPIException();
-            assert(mod->kind != Interactive_kind);
-            module = static_cast<AST_Module*>(cpythonToPystonAST(mod, fn));
-        } else {
-            module = pypa_parse(fn, inherited_flags);
-            RELEASE_ASSERT(module, "unknown parse error");
-        }
-
-        if (!cache_fp)
-            return std::vector<char>();
-
-        auto p = serializeAST(module, cache_fp);
-        checksum = p.second;
-        bytes_written += p.first;
-    } else {
-        RELEASE_ASSERT(!inherited_flags, "the old cpython parser doesn't support specifying initial future flags");
-        FILE* parser = popen(getParserCommandLine(fn).c_str(), "r");
-        char buf[80];
-        while (true) {
-            int nread = fread(buf, 1, 80, parser);
-            if (nread == 0)
-                break;
-            bytes_written += nread;
-
-            if (cache_fp)
-                fwrite(buf, 1, nread, cache_fp);
-            file_data.insert(file_data.end(), buf, buf + nread);
-
-            for (int i = 0; i < nread; i++) {
-                checksum ^= buf[i];
-            }
-        }
-        int code = pclose(parser);
-        assert(code == 0);
-    }
+    if (ENABLE_PYPA_PARSER || inherited_flags)
```

Comment:
I think we need an `else` after this, and we should also remove the `inherited_flags` check.